### PR TITLE
Accept single quotes #2

### PR DESCRIPTION
**Pull request by VanessaWhite26**

### CODE REVIEW

**Comment by VanessaWhite26** [comment]
[Parser: Accept single quotes](https://trello.com/c/WIoRNTPz/11-parser-accept-single-quotes)

File: wr-cli/parser.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        arr_user_string = user_input.split()
+        wrap_character= ""
+        stack = []
+        final_arr_user_string = []
+        for token in arr_user_string:
+            # keep the original token for comparisons, modified token for
+            # inserting into stack
+            modified_token = token.replace("\"", "").replace("\'", "")
+
+            if len(stack) != 0 and "\"" or "\'" in token:
```
Comment (edmudo) [inline wr-cli/parser.py]: I think you mean `if len(stack) != 0 and ("\"" in token or "\'" in token):`

File: wr-cli/parser.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                stack.append(modified_token)
+                final_token = " ".join(stack)
+                final_arr_user_string.append(final_token)
+                stack.clear()
+
+            elif len(stack) != 0 or ("\'" in token and token[1] != "\'"):
```
Comment (edmudo) [inline wr-cli/parser.py]: Should be `token[-1]`

File: wr-cli/parser.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            array of separated string tokens
+        """
+        arr_user_string = user_input.split()
+        wrap_character= ""
+        stack = []
+        final_arr_user_string = []
+        for token in arr_user_string:
+            # keep the original token for comparisons, modified token for
+            # inserting into stack
+            modified_token = token.replace("\"", "").replace("\'", "")
```
Comment (edmudo) [inline wr-cli/parser.py]: My bad. There's apparently a string function `strip()` that'll be better for this.

**Comment by edmudo** [comment]
[Parser: Accept  Mixed quotes](https://trello.com/c/WIoRNTPz/11-parser-accept-single-quotes)

File: wr-cli/parser.py [hunk 4]
```diff
@@ -106,15 +106,14 @@ def parse_input(self, user_input):
 
 
 def test():
```
Comment (Khfftwewy26) [inline wr-cli/parser.py:108]: Do we need to keep parser = Parser() to run the test? It is giving me an error.

**Pull request closed without merging**